homeassistant/components/climate/ecobee.py

```diff
@@ -204,6 +202,18 @@ def is_away_mode_on(self):
                 break
         return 'away' in mode
 
+    @property
```

Comment:
It's due to the mismatch between the ecobee module (always in F) and HA which is operating in C.

Do other modules have this limitation?  I don't have an others to test.  I suspect changing min_temp/max_temp to return F always would cause a problem with some.